diff --git a/Yaml.js b/Yaml.js
index 79362ce..da0400c 100644
--- a/Yaml.js
+++ b/Yaml.js
@@ -126,7 +126,7 @@ var YamlStringifyOptions = class extends Schema.Class("YamlStringifyOptions")({
 *
 * @public
 */
-var YamlParseError = class extends Schema.TaggedErrorClass()("YamlParseError", {
+var YamlParseError = class extends (Schema.TaggedErrorClass ?? Schema.TaggedError)()("YamlParseError", {
 	diagnostics: Schema.Array(YamlDiagnostic),
 	input: Schema.String
 }) {
@@ -144,7 +144,7 @@ var YamlParseError = class extends Schema.TaggedErrorClass()("YamlParseError", {
 *
 * @public
 */
-var YamlStringifyError = class extends Schema.TaggedErrorClass()("YamlStringifyError", {
+var YamlStringifyError = class extends (Schema.TaggedErrorClass ?? Schema.TaggedError)()("YamlStringifyError", {
 	diagnostics: Schema.Array(YamlDiagnostic),
 	value: Schema.Unknown
 }) {
diff --git a/YamlFormat.js b/YamlFormat.js
index 1dc01a9..5731ba9 100644
--- a/YamlFormat.js
+++ b/YamlFormat.js
@@ -47,7 +47,7 @@ var YamlFormattingOptions = class extends Schema.Class("YamlFormattingOptions")(
 *
 * @public
 */
-var YamlModificationError = class extends Schema.TaggedErrorClass()("YamlModificationError", {
+var YamlModificationError = class extends (Schema.TaggedErrorClass ?? Schema.TaggedError)()("YamlModificationError", {
 	path: Schema.Array(Schema.Union([Schema.String, Schema.Number])),
 	diagnostics: Schema.Array(YamlDiagnostic)
 }) {
